oamapps/postConfigure/installer.cpp

```diff
@@ -980,25 +977,6 @@ bool setOSFiles(string parentOAMModuleName, int serverTypeInstall)
 
             cmd = "cp " + installDir + "/local/etc/" + parentOAMModuleName + "/" + files[i] + ".calpont " + installDir + "/local/. > /dev/null 2>&1";
             system(cmd.c_str());
-        }
-        else
-        {
-            cmd = "sudo rm -f " + fileName + ".columnstoreSave";
-            system(cmd.c_str());
-
-            cmd = "sudo cp " + fileName + " " + fileName + ".columnstoreSave > /dev/null 2>&1";
-            system(cmd.c_str());
-
-            cmd = "sudo cat " + installDir + "/local/etc/" + parentOAMModuleName + "/" + files[i] + ".calpont >> " + fileName;
-
-            if (geteuid() == 0) system(cmd.c_str());
-
-            cmd = "sudo rm -f " + installDir + "/local/ " + files[i] + "*.calpont > /dev/null 2>&1";
-            system(cmd.c_str());
-
-            cmd = "sudo cp " + installDir + "/local/etc/" + parentOAMModuleName + "/" + files[i] + ".calpont " + installDir + "/local/. > /dev/null 2>&1";
-            system(cmd.c_str());
-        }
     }
```

Comment:
I'm not sure what's going on here but since files[] is empty we could get rid of it.